### PR TITLE
Init

### PR DESCRIPTION
**Pull request by paulopontesm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
 # ci-base
 
-Base Docker image
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/e9197451b4054234b142634a36cd096b)](https://www.codacy.com/project/Codacy/ci-base/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=codacy/ci-base&amp;utm_campaign=Badge_Grade_Dashboard)
+
```
Comment (rtfpessoa) [inline README.md]: Add docker version badge

File: README.md [hunk 2]
```diff
@@ -1,3 +1,27 @@
...
+Base docker image used by Codacy in Continuous Integration environments such as CircleCI.
+
+## Usage
+
+#### CircleCI
```
Comment (rtfpessoa) [inline README.md:10]: Command to build this?

**Review by rtfpessoa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rtfpessoa) [reply README.md]: @paulopontesm still missing

**Review by rtfpessoa — CHANGES_REQUESTED** [review]

**Comment by rtfpessoa** [comment]
Also seems like you do not have the keys to push. Check https://circleci.com/gh/codacy/ci-base/39?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,32 @@
+.DEFAULT-GOAL := help
+
+# Variables
+PROJECT_NAME?=$$(git remote -v | head -n1 | awk '{print $$2}' | sed 's/.*\///' | sed 's/\.git//')
+CWD=$(shell pwd)
+VERSION_NUMBER?=$$(docker run -v $(CWD):/repo codacy/ci-git-version:latest)## lazy. run with VERSION_NUMBER=x to prevent slow runs
```
Comment (rtfpessoa) [inline Makefile:6]: What does `lazy. run with VERSION_NUMBER=x to prevent slow runs` mean?

**Review by rtfpessoa — CHANGES_REQUESTED** [review]

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+end_of_line = lf
+insert_final_newline = true
+
+# Tab indentation (no size specified)
+[Makefile]
+indent_style = tab
+
+# Matches the exact files
+[{.circleci/config.yml}]
+indent_style = space
```
Comment (rtfpessoa) [inline .editorconfig]: I would put these as default in *

File: Makefile (hunk 3 above)
Comment (paulopontesm) [reply Makefile:6]: VERSION_NUMBER is a lazy variable (the ?= makes it lazy). The command is only evaluated if the VERSION_NUMBER is not defined.
Example:
```
$ make get-next-version-number VERSION_NUMBER=my_version
my_version
```

**Review by rtfpessoa — APPROVED** [review]

**Pull request merged**